enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -73,15 +72,15 @@ public class OnboardingService {
      * @param otpService OTP service.
      */
     @Autowired
-    public OnboardingService(
+    public OnboardingServiceImpl(
```

Comment:
Any better idea for name than `Impl`? Right now, we have `OnboardingServiceImpl` << `CommonOnboardingService` << `OnboardingService`

Reply:
I think it is ok.